### PR TITLE
RecipeBuilder and Kube methods for ranged item amounts

### PR DESCRIPTION
**Pull request by YoungOnionMC**

### CODE REVIEW

**Review by screret — COMMENTED** [review]
where is the input logic you said you'd add?

**Comment by YoungOnionMC** [comment]
> where is the input logic you said you'd add?

Are you able to write it in on this pr

**Comment by screret** [comment]
can't approve it if I do that

File: src/main/java/com/gregtechceu/gtceu/data/recipe/builder/GTRecipeBuilder.java [hunk 1]
```diff
@@ -321,17 +323,28 @@ public GTRecipeBuilder inputItems(MachineDefinition machine, int count) {
...
+        return inputItemsRanged(new ItemStack(output.get().asItem()), intProvider);
+    }
+
+    public GTRecipeBuilder inputItemsRanged(TagPrefix orePrefix, Material material, IntProvider intProvider) {
+        return inputItemsRanged(ChemicalHelper.get(orePrefix, material, 1), intProvider);
+    }
+
+    public GTRecipeBuilder inputItemsRanged(MachineDefinition machine, IntProvider intProvider) {
+        return inputItemsRanged(machine.asStack(), intProvider);
+    }
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/data/recipe/builder/GTRecipeBuilder.java]: remove these (they don't work, how would they even work?)

**Review by screret — CHANGES_REQUESTED** [review]

**Review by screret — APPROVED** [review]

**Pull request merged**